runhouse/servers/cluster_servlet.py

```diff
@@ -255,9 +242,35 @@ async def aperiodic_status_check(self):
                         # been called and completed.
                         await self.autostop_helper.register_activity_if_needed()
 
-                    if should_send_status_to_den:
+                    if should_send_status_and_logs_to_den:
                         cluster_rns_address = cluster_config.get("name")
-                        await rns_client.send_status(status, cluster_rns_address)
+                        cluster_uri = rns_client.format_rns_address(cluster_rns_address)
+                        api_server_url = status.cluster_config.get(
+                            "api_server_url", rns_client.api_server_url
+                        )
+
+                        sent_status = await rns_client.send_status(
+                            status=status,
+                            cluster_uri=cluster_uri,
+                            api_server_url=api_server_url,
+                        )
+
+                        if not sent_status:
```

Comment:
aren't we already surfacing the success/failure logs already for the status + logs (in `rns_client.py`)?